### DISCUSSION TITLE
How do you get ControlNet on Text Tab?

### DISCUSSION THREAD

**Discussion by HyunJae5463** [vladmandic/sdnext#3006] [General]
Dunno why it's so weird in SD.Next.  I just want https://github.com/Mikubill/sd-webui-controlnet inside the Text Tab. I can't install it because it says ControlNet is already installed but I don't want to use the SD.Next version though. 
![284446287-51172d20-606b-4b9f-aba5-db2f2417cb0b](https://github.com/vladmandic/automatic/assets/107355366/d438e16f-753b-49b4-aeb0-bd250ded57b7)

**Comment by brknsoul** [comment]
Either use the Control tab, or switch to original backend, and be restricted to SD15 models.
